ORK1Kit/ORK1Kit/Common/ORK1AnswerFormat.m

```diff
@@ -2910,17 +2918,9 @@ - (NSString *)stringForAnswer:(id)answer {
         if (self.useMetricSystem) {
             answerString = [NSString stringWithFormat:@"%@ %@", [formatter stringFromNumber:answer], ORK1LocalizedString(@"MEASURING_UNIT_KG", nil)];
         } else {
-            if (self.numericPrecision != ORK1NumericPrecisionHigh) {
-                double pounds = ORK1KilogramsToPounds(((NSNumber *)answer).doubleValue);
-                NSString *poundsString = [formatter stringFromNumber:@(pounds)];
-                answerString = [NSString stringWithFormat:@"%@ %@", poundsString, ORK1LocalizedString(@"MEASURING_UNIT_LB", nil)];
-            } else {
-                double pounds, ounces;
-                ORK1KilogramsToPoundsAndOunces(((NSNumber *)answer).doubleValue, &pounds, &ounces);
-                NSString *poundsString = [formatter stringFromNumber:@(pounds)];
-                NSString *ouncesString = [formatter stringFromNumber:@(ounces)];
-                answerString = [NSString stringWithFormat:@"%@ %@, %@ %@", poundsString, ORK1LocalizedString(@"MEASURING_UNIT_LB", nil), ouncesString, ORK1LocalizedString(@"MEASURING_UNIT_OZ", nil)];
-            }
+            double pounds = ORK1KilogramsToPounds(((NSNumber *)answer).doubleValue);
```

Comment:
The `pounds` value here will [always be rounded to a whole integer](https://github.com/CareEvolution/ResearchKit/blob/8cfb94763af40d5fa208f3598104a84c7ee89117/ORK1Kit/ORK1Kit/Common/ORK1Helpers_Internal.h#L328) — not what we want, right?